src/components/tooltip/Tooltip.tsx

```diff
@@ -88,6 +88,8 @@ export type TooltipProps = {
     children: React.ReactChild;
     /** A CSS class for the tooltip */
     className?: string;
+    /** Component maintain focus within the tooltip until escaped */
+    focusTrapComponent?: React.ElementType;
```

Comment:
not sure I understand this part, is the focused trapped within the tooltip? Or is it trapped within this component called `focusTrapComponent`?